src/sqs/sqs.ts

```diff
@@ -82,8 +82,9 @@ export class SqsQueue<MessagePayload> {
 
 		const sendMessageResult = await this.sqs.sendMessage(params)
 			.promise();
-		logger.info(`Successfully added message to sqs queue messageId: ${sendMessageResult.MessageId}`);
+		logger.info({ DelaySeconds: delaySec, MessageId: sendMessageResult.MessageId }, `Successfully added message to sqs queue messageId: ${sendMessageResult.MessageId}`);
```

Comment:
nit: I think the convention is to use camelCase (starting with lower letter) for the properties
Also, probably I'd opted for something more verbose for the message ID (newMessageId)

Reply:
fixed.